cliff.toml

```diff
@@ -58,6 +58,7 @@ split_commits = false
 commit_parsers = [
   { message = "^ci:", skip = true },
   { message = "^.*\\(android\\):", skip = true },
+  { message = "^.*\\(gradle\\):", skip = true },
```

Comment:
@abhaysood Same question as before. Why should gradle be first class citizen as a scope. It's kinda under "android" no? If we want to make `gradle` a first class scope, then all commits related to gradle should also contain that scope. But I don't see the need for this, hence asking.

Reply:
We need a different scope so that gradle plugin can get it's own changelog as it's a different artefact and has it's own release cycle.

Reply:
Here, by gradle we mean the `gradle-plugin`. We could rename the scope to be more verbose, although I do prefer it to be a single word.

Reply:
hmm. I get that, but wouldn't it be easier for a user to just read all android related changes for the "android" release. gradle will have it's own release cycle, but won't have it's own changelog. just a suggestion thinking from a user's point of view. it's easier to get all changelog at one place. up to you.

Reply:
Separating the Android SDK and Gradle plugin is actually clearer due to their distinct scopes. The Android SDK is a runtime library, while the Gradle plugin is a combination of build-time scripts. The Gradle plugin's utility extends beyond native Android development, as will also be used for hybrid apps like Flutter and React Native.

Furthermore, the setup and testing procedures for upgrading each component differ significantly.